src/sentry/api/serializers/models/user.py

```diff
@@ -144,7 +144,9 @@ def _get_identities(
             results[item.user_id].append(item)
         return results
 
-    def get_attrs(self, item_list: Sequence[User], user: User) -> MutableMapping[User, Any]:
+    def get_attrs(
+        self, item_list: Sequence[User], user: User, **kwargs: Any
```

Comment:
I'd prefer the type ignore for override than making these signatures overly broad.  or at least raise a `TypeError` if unexpected kwargs are passed

tbh this is "OOP kinda bad" rearing its head

Reply:
gotcha. yeah i was feeling icky about that. May also be able to use some kind of generics on the base class to help with this. i'll iterate on this a bit when i get time.

Reply:
(ah your comment was unrelated to the generic stuff i was thinking about)